src/frontend/screens/Settings/ProjectConfig.tsx

```diff
@@ -86,6 +86,12 @@ const ProjectConfig = () => {
     ]);
   }, [didError, t]);
 
+  React.useEffect(() => {
+    config.practiceModeBarToggle!(false);
+
+    return () => config.practiceModeBarToggle!(true);
+  }, []);
```

Comment:
think it would make more sense for the `PracticeMode` component to handle this toggled state, instead of having individual screens handle it. will comment in that component to explain